jobbergateapi2/apps/applications/routers.py

```diff
@@ -48,7 +74,7 @@ async def applications_create(
             application_created_id = await database.execute(query=query, values=values)
 
         except INTEGRITY_CHECK_EXCEPTIONS as e:
-            raise HTTPException(status_code=422, detail=str(e))
+            raise HTTPException(status_code=status.HTTP_409_CONFLICT, detail=str(e))
     application_location = f"{settings.S3_BASE_PATH}/{application.application_owner_id}/applications/{application_created_id}/jobbergate.tar.gz"  # noqa
```

Comment:
It is indeed, but since it is the line to configure the location of the application, it will be ugly if we break that. There is already a noqa in the line to indicate that to the linter. It would be a problem if we have a lot of them, but there is only here.